src/lib/ChatWindow.vue

```diff
@@ -72,6 +75,7 @@
 				:username-options="usernameOptionsCasted"
 				:emoji-data-source="emojiDataSource"
         :attachment-options="attachmentOptionsCasted"
+        :call="callCasted"
```

Comment:
`callCasted` parece ser um pouco generico demais. Acho que deixando explicito o tipo dele fica mais legivel
```suggestion
        :call="callObject"
```

Reply:
Concordo que `casted` não é muito bom. Porém o próprio vac utiliza a nomenclatura em todas as outras props. Aí não sei se vale a pena termos somente uma nomeada "bonitinho". Preferi manter o padrão (mesmo que não tão bom :sob:) mas podemos discutir alterar sem problemas.